sensor_test.py

```diff
@@ -202,5 +201,64 @@ def test_update_sensor_firmware(get_sensor_info, update_sensor_firmware):
     sensor_info_after_update = get_sensor_info()
     current_firmware_version = sensor_info_after_update.firmware_version
     assert (
-        current_firmware_version == max_firmware_version
+            current_firmware_version == max_firmware_version
     ), "Sensor firmware version not max"
+
+
+def test_set_empty_sensor_name(get_sensor_info, set_sensor_name):
+    """
+    Test Steps:
+    1. Get original sensor name.
+    2. Set sensor name to an empty string.
+    3. Validate that sensor responds with an error.
+    4. Get current sensor name.
+    5. Validate that sensor name didn't change.
+    """
+
+    log.info("1. Get original sensor name")
+    sensor_info = get_sensor_info()
+    original_name = sensor_info.name
+
+    log.info("2. Set sensor name to an empty string")
+    empty_name = ""
+    response_to_empty_name = set_sensor_name(empty_name)
+
+    log.info("3. Validate that sensor responds with an error")
+    assert response_to_empty_name == "error", "Sensor didn't respond with an error"
```

Comment:
Directly relating to the first comment: here's how the check would look like if you keep `_make_valid_request()` unchanged:
```
assert response_to_empty_name =={}, "Sensor didn't respond with an error"
```